src/webportal/src/app/home/home/virtual-cluster-statistics.jsx

```diff
@@ -101,8 +101,8 @@ const vcListColumns = [
     isResizable: true,
     onRender(vc) {
       const { resourcesUsed, resourcesTotal } = vc;
-      const resourceAvailable = vc.resourceAvailable || resourcesTotal;
-      const insufficient = !(resourceAvailable.GPUs === resourcesTotal.GPUs);
+      const resourcesGuaranteed = vc.resourcesGuaranteed || resourcesTotal;
```

Comment:
>vc.resourcesGuaranteed || resourcesTotal; [](start = 34, length = 41)

In which case, the resourcesGuaranteed is null?

Reply:
old version api